packages/connectkit/src/components/Common/Modal/index.tsx

```diff
@@ -514,7 +514,7 @@ const Modal: React.FC<ModalProps> = ({
                           delay: mobile ? 0.01 : 0,
                         }}
                       >
-                        <ProfileIcon signedIn={!!signedIn} />
+                        <ProfileIcon signedIn={!!isSignedIn} />
```

Comment:
Instead of forcing this to a boolean the `isSignedIn` is already typed as a boolean so `signedIn={isSignedIn}` should do the same thing here and is more clear to me.

Also I'd recommend changing the prop for ProfileIcon to `isSignedIn` also just to be clear thats a boolean by using the `is` prefix on the variable.

Reply:
Good notes! Updated 🫡